homeassistant/components/homematic/const.py

```diff
@@ -152,7 +152,7 @@
         "IPRemoteMotionV2",
         "IPWInputDevice",
     ],
-    DISCOVER_COVER: ["Blind", "KeyBlind", "IPKeyBlind", "IPKeyBlindTilt", "IPGarage"],
+    DISCOVER_COVER: ["Blind", "KeyBlind", "IPKeyBlind", "IPKeyBlindTilt", "IPGarage", "IPKeyBlindMulti"],
```

Comment:
```suggestion
    
    DISCOVER_COVER: [
        "Blind",
        "KeyBlind",
        "IPKeyBlind",
        "IPKeyBlindTilt",
        "IPGarage",
        "IPKeyBlindMulti",
    ],
```

Reply:
This should fix it.

Reply:
That works. Perfect and thanks! I tried to install black on Raspbian. But that did not work. The manual edit now did the trick.